proposed/2021/DotnetNuGetSign-Verbosity.md

```diff
@@ -0,0 +1,102 @@
+# `dotnet nuget sign --verbosity`
+
+* Status: **In Review**
+* Author: [Heng Liu](https://github.com/heng-liu)
+* Issue: [#11173](https://github.com/NuGet/Home/issues/11173) - No certificate info shows when running dotnet nuget sign on default verbosity
+
+## Problem background
+
+`dotnet nuget sign` command doesn't provide any output by default due to logical error in the implementation. If the user didn't pass value for `--verbosity` option, then the log level is set to [`LogLevel.Minimal`](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L58). Customers need to know the results of signing.
+
+
+## Who are the customers
+
+Package consumers that use the `dotnet nuget sign` command to sign package.
+
+## Goals
+
+* Change the output of `dotnet nuget sign` command for various `--verbosity` options.
+
+## Solution
+
+The details that should be displayed on each verbosity level are described below. Each level should display the same as the level below plus whatever is specified in that level. In that sense, `quiet` will be give the least amount of information, while `diagnostic` the most.
```

Comment:
I assume each level maps to `nuget sign -Verbosity [normal|quiet|detailed]` with the added `minimal` &`diagnostic` levels within dotnet CLI.

Reply:
Yes. Added `Comparison` section to make it more clear :)